<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selfun</class>
 <widget class="QWidget" name="selfun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>selfun</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确认(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/ok</normaloff>:/images/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>放弃(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skdraw.qrc"/>
 </resources>
 <connections/>
</ui>
